Arrowgene.Ddon.Shared/Model/MailType.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Arrowgene.Ddon.Shared.Model
+{
+    public enum MailType : uint
```

Comment:
they use a whole ass uint for this? 😭

Reply:
It looks like it is actually an `int` but yes, it seems they wasted 4 bytes.

```c++
enum nNetSv::E_MAIL_TYPE : __int32
{
  MAIL_TYPE_USER = 0x1,
  MAIL_TYPE_OPERATION = 0x2,
};
```